src/components/describe/describeWebview.ts

```diff
@@ -0,0 +1,91 @@
+import * as vscode from 'vscode';
+
+export class DescribePanel {
+	public static readonly viewType = 'vscodeKubernetesDescribe';
+	public static currentPanels = [];
+
+	private readonly panel: vscode.WebviewPanel;
+    private readonly extensionPath: string;
```

Comment:
Something wonky in the indentation here?

Reply:
fixed (I hope)